Assignment/Assignment-1.ipynb

```diff
@@ -1097,30 +1282,57 @@
+     ]
+    }
+   ],
    "source": [
-    "pickle.dump(labels, open( \"labels.p\", \"wb\" ) )\n",
-    "dialogues.close()"
```

Comment:
The file that has been open using `open` must be closed. You should essentially not delete these 2 lines. Besides you are missing out on labels.p . Did any error occurred?